README.rst

```diff
@@ -11,7 +11,7 @@ browsers.
 This is accomplished by providing a HTML/JavaScript GUI for entering source
 code and visualization of output, and a server part which evaluates the
 requested source code. Note that this shell is not scalable and it uses
-only one instance on GAE, thus all evaluation requests are queued and it
+only one instance on GAE(Google App Engine),thus all evaluation requests are queued and it
```

Comment:
IMO it would be better to add the GAE acronym in parenthesis after the first instance of the expression "Google App Engine", in line 8 above.

Reply:
Ok sure I will do it @waldyrious (Thanks for your suggestion).

Reply:
@waldyrious Why always my test fail everytime?

Reply:
I have no idea, sorry :(